### DISCUSSION TITLE
4.0.0.dev10 considered stable?

### DISCUSSION THREAD

**Discussion by neutric** [opensteno/plover#1499] [Technical Discussion]
Do you consider 4.0.0.dev10 the current stable version of Plover? This question came up in https://github.com/Homebrew/homebrew-cask/pull/120183. Thanks!

**Comment by JoshuaGrams** [comment]
The devs have a [checklist](https://github.com/openstenoproject/plover/issues/1206) of things before they're willing to officially make a stable 4.0 release? And they're both busy people so who knows when that'll happen. But dev10 is de facto stable: nearly everyone in the community has been using the 4.0 dev versions for at least two years now. Looks like 3.1.1 is five years old as of yesterday...

**Reply by neutric** [reply]
@JoshuaGrams Thanks for your quick reply!

**Comment by L4gl** [comment]
I would say it is stable. I have used it for a bit and I can say for certain it is good. [There is a checklist](https://github.com/openstenoproject/plover/issues/1206) of things for the dev's to add.
